ui/src/app/pages/admin/admin-institution-edit.tsx

```diff
@@ -154,44 +165,72 @@ export const AdminInstitutionEdit = withUrlParams()(class extends React.Componen
     return;
   }
 
+  setTierRequirement(membershipRequirement, attribute) {
+    // TODO(RW-6933): Implement new institution admin UI with CT support.
+    this.setState(fp.set(['institution', attribute],
+      [{accessTierShortName: AccessTierShortNames.Registered,
+        membershipRequirement: membershipRequirement.value, eRARequired: true}]));
+
+    // Dual write DuaTypeEnum for now. Because account creation flow still uses that field.
+    const duaType = membershipRequirement.value === InstitutionMembershipRequirement.ADDRESSES
+        ? DuaType.RESTRICTED : DuaType.MASTER;
+    this.setState(fp.set(['institution', 'duaTypeEnum'], duaType));
+  }
+
   setEmails(emailInput, attribute) {
     const emailList = emailInput.split(/[,\n]+/);
-    this.setState(fp.set(['institution', attribute], emailList.map(email => email.trim())));
+    // For now, only RT requirement is supported, so fine to set tierEmailAddresses to an single element array.
+    this.setState(fp.set(['institution', attribute],
+        [{accessTierShortName: AccessTierShortNames.Registered, emailAddresses: emailList.map(email => email.trim())}]));
+  }
+
+  setDomains(emailInput, attribute) {
+    const emailList = emailInput.split(/[,\n]+/);
+    // For now, only RT requirement is supported, so fine to set tierEmailAddresses to an single element array.
+    this.setState(fp.set(['institution', attribute],
+        [{accessTierShortName: AccessTierShortNames.Registered, emailDomains: emailList.map(email => email.trim())}]));
   }
 
   // Check if the fields have not been edited
-  fieldsNotEdited() {
+  fieldsEdited() {
     return (this.isAddInstitutionMode && !this.fieldsNotEditedAddInstitution)
         || (institutionToEdit && this.fieldsNotEditedEditInstitution);
   }
 
   get fieldsNotEditedAddInstitution() {
     const {institution} = this.state;
     return institution.displayName || institution.userInstructions ||
-      institution.organizationTypeEnum || institution.duaTypeEnum || institution.emailAddresses || institution.emailDomains;
+      institution.organizationTypeEnum || institution.tierRequirements
+        || institution.duaTypeEnum || institution.tierEmailDomains
+        || institution.tierEmailAddresses;
   }
 
   get fieldsNotEditedEditInstitution() {
     const {institution} = this.state;
     return institution.displayName === institutionToEdit.displayName &&
```

Comment:
FYI: Existing issue, but this pattern may be a lot more readable here for handling partial comparisons:

https://github.com/all-of-us/workbench/blob/78528ce66e0f000f63fb2b42b2e2c6392f88347b/ui/src/app/pages/analysis/runtime-panel.tsx#L161-L168